### PR TITLE
upgrade deps to work with latest chai version

### PR DESCRIPTION
**Pull request by jenius**
Just ran the tests and everything is still looking good :+1: 

I updated the chai dev dependency to "latest", as there's no way for this to break production code, it will just fail your tests as soon as there is a breaking change pushed, which is more or less always helpful, since this is something you definitely need to know.

### CODE REVIEW

**Comment by stefanwalther** [comment]
+1

**Comment by jenius** [comment]
Ping?

**Comment by reker-** [comment]
Great. I'm just going to depend on your repo for the time being.

**Comment by DavertMik** [comment]
Ping-ping!

**Comment by yoannmoinet** [comment]
Ping !

**Comment by stefanwalther** [comment]
:+1:

**Comment by epicmiller** [comment]
Bump. @Bartvds, can we get this pulled in?..

**Comment by mearns** [comment]
Related to Issue #10

**Comment by Bartvds** [comment]
Please see my comment https://github.com/Bartvds/chai-fs/issues/10#issuecomment-207043538 

tl;dr: someone responsible should take over the module and get you all back on track. Please coordinate a nominee in #10

**Comment by tomasmck** [comment]
@keithamus when could this be merged?
